scripts/utils.sh

```diff
@@ -51,29 +50,24 @@ function spawn_port_forwarding_command() {
         fi
     fi
 
-    cat <<EOF | sudo tee "${socket_file}"
```

Comment:
We don't need to socket file now ?

Reply:
no it just a single systemd file with the socket config inside of it 
make the config and managing it more simple